Aliucord/src/main/java/com/aliucord/coreplugins/badges/Utils.kt

```diff
@@ -0,0 +1,38 @@
+package com.aliucord.coreplugins.badges
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.widget.ImageView
+import com.aliucord.*
+
+private val imageCache = HashMap<String, Bitmap>()
+
+/**
+ * Set a remote image to be displayed in this view.
+ * This will be cached in memory for the lifecycle of the application.
+ * As such, this should only be used for long-living and often used images.
+ */
+internal fun ImageView.setCacheableImage(url: String) {
```

Comment:
this seems like a bad idea, do we really need this?

you should at least add a timeout, eg cache for 10 minutes then clear

Reply:
Why? It is not like they're going to change as url includes their hash, and I don't think it is going to use a lot of memory.

Reply:
This is exactly what was done before
Also in the rare event someone actually sees a badge it's likely they're going to see it again soon enough.
I just don't see a point in adding complexity for something that's never going to be an issue.